.github/CODEOWNERS

```
@@ -113,6 +113,8 @@
 /modules/programs/newsboat.nix                        @sumnerevans
 /tests/modules/programs/newsboat                      @sumnerevans
 
+/modules/programs/nix-index.nix                       @ambroisie
+
```

Please add yourself as codeowner for the tests as well:

```suggestion
/tests/modules/programs/nix-index

```